take_screenshot.py

```diff
@@ -0,0 +1,36 @@
+import os
+from android_device import AndroidDevice
+from PIL import Image, ImageDraw
+
+# Find android device
+device = AndroidDevice('emulator-5554')
```

Comment:
From my first comment, there should be a way to loop through the devices listed by 'adb devices' and find the one which is running the game.
So this part of the code can become more 'dynamic'.